apps/deploy-web/src/components/deployments/ManifestUpdate.tsx

```diff
@@ -19,49 +19,48 @@ import { DeploymentDto, LeaseDto } from "@src/types/deployment";
 import { ApiProviderList } from "@src/types/provider";
 import { AnalyticsEvents } from "@src/utils/analytics";
 import { deploymentData } from "@src/utils/deploymentData";
-import { getDeploymentLocalData, saveDeploymentManifest } from "@src/utils/deploymentLocalDataUtils";
+import { saveDeploymentManifest } from "@src/utils/deploymentLocalDataUtils";
 import { sendManifestToProvider } from "@src/utils/deploymentUtils";
 import { TransactionMessageData } from "@src/utils/TransactionMessageData";
+import RemoteDeployUpdate from "../remote-deploy/update/RemoteDeployUpdate";
 import { ManifestErrorSnackbar } from "../shared/ManifestErrorSnackbar";
 import { Title } from "../shared/Title";
 
 type Props = {
   deployment: DeploymentDto;
   leases: LeaseDto[];
   closeManifestEditor: () => void;
+  remoteDeploy: boolean;
+  showOutsideDeploymentMessage: boolean;
+  editedManifest: string;
+  deploymentVersion: string | null;
+  setDeploymentVersion: (value: React.SetStateAction<string | null>) => void;
```

Comment:
There's a type from react for such functions
```suggestion
  setDeploymentVersion: Dispatch<SetStateAction<string | null>>;
```